pyxrf/model/fileio.py

```diff
@@ -737,12 +738,14 @@ def read_hdf_APS(working_directory,
     ----------
     working_directory : str
         path folder
-    file_names : list
-        list of chosen files
+    file_names : str
+        selected h5 file
     channel_num : int, optional
         detector channel number
     spectrum_cut : int, optional
         only use spectrum from, say 0, 3000
```

Comment:
so this is the upper end of the spectrum to use?  Is there any reason you would want to be able to select a range that starts at a number greater than 0?